public/app/features/plugins/admin/components/PluginScreenshotCarousel.tsx

```diff
@@ -0,0 +1,22 @@
+import { config } from '@grafana/runtime';
+import { Carousel } from '@grafana/ui';
+
+import { CatalogPlugin, Screenshots } from '../types';
+
+const buildScreenshotPath = (plugin: CatalogPlugin, path: string) => {
+  return `${config.appSubUrl}/api/gnet/plugins/${plugin.id}/versions/${plugin.latestVersion}/images/${path}`;
+};
+
+interface PluginScreenshotCarouselProps {
+  plugin: CatalogPlugin;
+  screenshots: Screenshots[];
+}
+
+export const PluginScreenshotCarousel: React.FC<PluginScreenshotCarouselProps> = ({ screenshots, plugin }) => {
```

Comment:
This components only function seems to be parsing props for the carousel, could this better be done where this component is called and remove it?